runtime/marshaler.go

```diff
@@ -15,8 +15,8 @@ type Marshaler interface {
 	NewDecoder(r io.Reader) Decoder
 	// NewEncoder returns an Encoder which writes bytes sequence into "w".
 	NewEncoder(w io.Writer) Encoder
-	// ContentType returns the Content-Type which this marshaler is responsible for.
-	ContentType() string
+	// ContentType returns the response Content-Type for "v".
```

Comment:
Can you elaborate on what "v" would be?

Reply:
v is the same thing that `Marshal(v interface{})` is passed, I'm honestly just following convention here. should I update comments on `Marshal` as well? (happy to do that just lmk)

side note, I am not actually sure why all of these interfaces are not instead of type `proto.Message` which is what is actually being passed...